torchvision/models/mobilenetv3.py

```diff
@@ -0,0 +1,291 @@
+import torch
+
+from functools import partial
+from torch import nn, Tensor
+from torch.nn import functional as F
+from typing import Any, Callable, List, Optional
+
+from torchvision.models.utils import load_state_dict_from_url
+from torchvision.models.mobilenetv2 import _make_divisible, ConvBNActivation
+
+
+__all__ = ["MobileNetV3", "mobilenet_v3_large", "mobilenet_v3_small"]
+
+
+# TODO: add pretrained
+model_urls = {
+    "mobilenet_v3_large_1_0": None,
+    "mobilenet_v3_small_1_0": None,
+}
+
+
+class _InplaceActivation(nn.Module):
+
+    def __init__(self, inplace: bool = False):
+        super().__init__()
+        self.inplace = inplace
+
+    def extra_repr(self) -> str:
+        return 'inplace=True' if self.inplace else ''
+
+
+class Identity(_InplaceActivation):
+
+    def forward(self, input: Tensor) -> Tensor:
+        return input
+
+
+def hard_sigmoid(x: Tensor, inplace: bool = False) -> Tensor:
+    return F.relu6(x + 3.0, inplace=inplace) / 6.0
```

Comment:
Not really fully inplace but not sure it's worth it. If we want this perhaps replace it with something like:

```python
def hard_sigmoid(input: Tensor, inplace: bool = False) -> Tensor:
    if inplace:
        input.add_(3.0)
        F.relu6(input, inplace=True)
        input.div_(6.0)
        return input
    else:
        return F.relu6(input + 3.0, inplace=False) / 6.0
```

Reply:
Hi @datumbox , I have a question why we don't use [`nn.Hardswitch`](https://pytorch.org/docs/master/generated/torch.nn.Hardswish.html#torch.nn.Hardswish) here.

PS. I tested `nn.Hardswitch` that it can be jit scriptable, and it should be able to export on onnx backend after this [PR](https://github.com/pytorch/pytorch/pull/48423) is done, (but I didn't test this now).

Reply:
@zhiqwang I just missed it, thanks for pointing this out! I've replaced it with the built-in one.